### PR TITLE
Fix relative link "sync the contents" to configlet doc

### PR DESCRIPTION
**Pull request by angelikatyborska**

### CODE REVIEW

**Review by ee7 — APPROVED** [review]
For a later PR, we should:
- Rename the file at [building/configlet/generating-documents.md](https://github.com/exercism/docs/blob/main/building/configlet/generating-documents.md) (or move the contents to [building/configlet/sync.md](https://github.com/exercism/docs/blob/main/building/configlet/sync.md))
- Avoid the word "generate" when talking about syncing. All these docs were written before we had `configlet generate`, but they're now likely to cause confusion.

**Comment by iHiD** [comment]
Thanks for this. 

A key thing to understand about this repo is that its intended to power the website, rather than being correct in itself. So the links need to work in the context of the website, not in relative context within the repo. This means specifically that they need to work with the rendered URLs as per the `config.json` files.

So, the correct link for this page is https://exercism.lol/docs/building/configlet/generating-documents - so I think the correct link in the document should be `(../configlet/generating-documents)` (without the `.md`). Atm we don't have any of the `#` links working on the website, but these should get automatically updates, so you can probably safely leave that bit in and we can get those working in the near-future.

Does that make sense?

File: building/tracks/practice-exercises.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ exercises
 
 **Presence:** Required if the exercise implements a Problem Specifications Exercise with an `introduction.md` file
 
-If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `introduction.md` file. configlet has functionality to automatically (sync the contents)[./configlet/generating-documents.md#documentpracticeexercisesintroductionmdfile] of this file.
+If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `introduction.md` file. configlet has functionality to automatically [sync the contents](../configlet/generating-documents.md#document-practice-exercises-introductionmd-file) of this file.
```
Comment (ee7) [inline building/tracks/practice-exercises.md]:
```suggestion
If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `introduction.md` file. configlet has functionality to automatically [sync the contents](../configlet/generating-documents#document-practice-exercises-introductionmd-file) of this file.
```

File: building/tracks/practice-exercises.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ As part of his teenage rebellion, Bob has decided to only communicate using ASCI
 
 **Presence:** Required
 
-If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `instructions.md` file (or `description.md` file if there is no `instructions.md` file). configlet has functionality to automatically (sync the contents)[./configlet/generating-documents.md#documentpracticeexercisesinstructionsmdfile] of this file.
+If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `instructions.md` file (or `description.md` file if there is no `instructions.md` file). configlet has functionality to automatically [sync the contents](../configlet/generating-documents.md#document-practice-exercises-instructionsmd-file) of this file.
```
Comment (ee7) [inline building/tracks/practice-exercises.md]:
```suggestion
If the exercise implements a [Problem Specifications Exercise](https://github.com/exercism/problem-specifications/), this file's contents should match the Problem Specification Exercise's `instructions.md` file (or `description.md` file if there is no `instructions.md` file). configlet has functionality to automatically [sync the contents](../configlet/generating-documents#document-practice-exercises-instructionsmd-file) of this file.
```

**Review by ee7 — COMMENTED** [review]
Seems like Jeremy is suggesting this.

**Comment by angelikatyborska** [comment]
Got it!

**Review by ee7 — APPROVED** [review]
LGTM.

**Pull request merged**